web-frontend/src/pages/Assets/AssetImportModal/ImportResult.tsx

```diff
@@ -0,0 +1,48 @@
+import { FiCheck, FiX } from 'react-icons/fi';
+
+import { InputsErrors } from '@/const/assets/assets';
+import Intl from '@/i18n/i18n';
+import { Asset } from '@/models/AssetModel';
+import { maskAddress } from '@/utils';
+
+interface ImportResultProps {
+  closeModal: () => void;
+  data: Asset;
+  importResult: boolean;
+  inputError: InputsErrors | null;
+}
+
+export function ImportResult(props: ImportResultProps) {
```

Comment:
I believe we have an anti patter here in terms of UI. We should sync with Lucas/Ena about it!

Reply:
@pivilartisant We agreed to go with toast for now. Mario will sync with you.

Reply:
Hey Paul, just synced with Ena about it. Let's keep as it is, with Toast messages (with the good information, of course) and we'll sync with Lucas to find a solution for that.